packages/python/plotly/plotly/io/_json.py

```diff
@@ -57,6 +57,25 @@ def coerce_to_strict(const):
         return const
 
 
+_swap_json = (
+    ("<", "\\u003c"),
+    (">", "\\u003e"),
+    ("/", "\\u002f"),
+)
+_swap_orjson = _swap_json + (
+    ("\u2028", "\\u2028"),
+    ("\u2029", "\\u2029"),
```

Comment:
These are apparently JavaScript line terminator characters. The JS parser will see this end-of-line, see that the data structure is incomplete, and throw an error. AFAICT this isn't in itself a security issue, but it's a bug - if you ever wanted these characters in your figure for real it wouldn't work (when inserted in HTML).

The standard library `json` converts at least these unicode characters into escape sequences anyway, so we don't need to worry about them, but `orjson` sends them as unicode characters.